modules/virt-cloning-pvc-of-vm-disk-into-new-datavolume.adoc

```diff
@@ -48,36 +42,22 @@ For example:
 apiVersion: cdi.kubevirt.io/v1beta1
 kind: DataVolume
 metadata:
-  name: <cloner-datavolume> <1>
+  name: <datavolume> <1>
 spec:
   source:
     pvc:
-      namespace: "<source-namespace>" <2>
-      name: "<my-favorite-vm-disk>" <3>
-  pvc:
-    accessModes:
-      - ReadWriteOnce
-    resources:
-      requests:
-        storage: <2Gi> <4>
-ifdef::blockstorage[]
-    volumeMode: Block <5>
-endif::[]
+      namespace: "<source_namespace>" <2>
+      name: "<my_vm_disk>" <3>
```

Comment:
You will still need the `storage` section, even if it is blank it will look like this:

```yaml
  storage: {}
```
Like that it will automatically populate all the required information if we can detect it. If you want to specify the storage class it needs to look like this:

```yaml
  storage:
     storageClassName: <storage class name>
```